README.md

```diff
@@ -15,6 +15,7 @@ It contains auto-configurations for Spring Boot which will instrument and trace
 * Zuul
 * RxJava
 * Standard logging - logs are added to active span
+* Spring Messaging
```

Comment:
Could you please add more text here - e.g. wht exactly is supported - kafka,  rabbitmq?

Reply:
just to sell it better :)